src/layer/const.ts

```diff
@@ -149,6 +149,8 @@ export type ImageProperties = {
 
 export const SH_SERVICE_HOSTNAMES_V3: string[] = [
   'https://services.sentinel-hub.com/',
+  'https://staging1-awseu.sentinel-hub.com/',
+  'https://staging0-awseu.sentinel-hub.com/',
```

Comment:
Will check with the team if it's ok to have these staging URLs public.

Reply:
In general the `registerHostnameReplacing(originalHostname, desiredHostname)` could be used to "override" the all urls with `originalHostname` with the `desiredHostname`, but the underlying `replaceHostnamesInterceptor` doesn't seem to work properly - checking.

Reply:
Agree, removed those extra urls and let registerHostnameReplacing handle its work.